dynamic-layers/imx708-layer/recipes-kernel/linux/linux-raspberrypi-dev.bbappend

```diff
@@ -0,0 +1,4 @@
+SRC_URI += "file://imx708-overlay.dts;subdir=git/arch/${ARCH}/boot/dts/overlays"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
```

Comment:
I'm not sure what this is all about in the imx708 layer. Shouldn't this be a bbappend in our kernel recipes? Also, isn't this upstreamable?